script/Ciel/Day 7/07_01_CIEL07_7.txt

```diff
@@ -468,7 +469,7 @@ Mere seconds ago, a web of over seven hundred spirit strings had surrounded the
 [sha:9c6396c742c0f93d8a4e286822589d66cb360cec]{
 -- Page 614, Offset 16283.
 -- 「……魔術詠唱はなかったな。指先から魔力を流しただけで燃やしやがったか。嘘みてぇな生成量だ。並の魔術師の何百倍だ？　とっくに化け物だな代行者シエル？
-"I didn't hear any Magecraft Incantation...
+"I didn't hear any %{i}Magecraft Incantation%{/i}...
```

Comment:
arc route already has roa mentioning automatic incantation so I don't think this needs italics